### PR TITLE
[1317] UI for Roles - Roles Tab [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by yasha-dev1**
Roles Module, roles tab

**Checklist**
- [x] My code follows the style guidelines of this project
- [x] My commits follow the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: admin/src/api/fetchCapabilitiesList.js
Comment (jremen) [inline admin/src/api/fetchCapabilitiesList.js:1]: Please use getFetch function from fetching.js instead, delete this

File: admin/src/api/fetchUsers.js
Comment (jremen) [inline admin/src/api/fetchUsers.js:1]: Use getFetch from fetching.js

File: admin/src/api/fetching.js
Comment (jremen) [inline admin/src/api/fetching.js:1]: avoid resaving all existing files with complete rewrite of code formatting. Causes a mess in review process

File: admin/src/api/lisOfRoles.js
Comment (jremen) [inline admin/src/api/lisOfRoles.js:1]: Use getFetch and postFetch, remove this file completely

**Review by jremen — CHANGES_REQUESTED** [review]
Avoid rewrite of the files you have not changed at all (by changing formatting).
Do not create custom fetch methods with preset values, use getFetch and postFetch methods with variables instead. Remove created files

File: admin/src/components/ActionsBar.jsx
Comment (ihdk) [inline admin/src/components/ActionsBar.jsx:1]: use localization functions __() for all translatable texts, please

File: admin/src/components/ActionsBar.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        liveUpdate
+        autoFocus
+        onChange={(input) => handleFiltering({ input, type: 'search' })}
+        placeholder={__('Search')}
+      />
+      <Button className={'simple'}>
+        <PlusIcon /> Add capability
+      </Button>
+
+      <p style={{ marginLeft: 'auto' }}>
```
Comment (ihdk) [inline admin/src/components/ActionsBar.jsx:23]: avoid inline styling if not really necessary

File: admin/src/components/SelectedCapabilities.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+export default function SelectedCapabilities(props) {
+  const { selecteditems, toggleSelectItem } = props
```
Comment (ihdk) [inline admin/src/components/SelectedCapabilities.jsx:2]: use please camelCase for variables to keep better code readability

File: admin/src/components/SelectedCapabilities.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  return (
+    <div className="urlslab-selected-capabilities">
+      <h5 style={{ marginBottom: '15px' }}>selected</h5>
+      {selecteditems.map((item) => (
+        <div className="urlslab-capability-item">
+          <label role="Controlled Checkbox" className={`urlslab-checkbox `}>
```
Comment (ihdk) [inline admin/src/components/SelectedCapabilities.jsx:8]: Simple class names doesn't need to be wrapped in brackets.
Use simple quotes `""` if not used variables inside string.

File: admin/vite.config.js
Comment (jremen) [inline admin/vite.config.js:1]: Why is this changed? Please beware!!!

File: admin/vite.config.development.js
Comment (jremen) [inline admin/vite.config.development.js:1]: Why this?

File: admin/src/components/SelectedCapabilities.jsx
Comment (jremen) [inline admin/src/components/SelectedCapabilities.jsx:1]: why this when we have MultiSelectMenu ?

File: admin/src/components/ModuleViewHeader.jsx
Comment (jremen) [inline admin/src/components/ModuleViewHeader.jsx:1]: Why all changes to formatting?! Please use ESLint with predefined rules!!!
